### PR TITLE
Testacular part3

### PR DESCRIPTION
**Pull request by rperez20**
please check my code (soon) =P

### CODE REVIEW

**Review by stefaniaoverman — COMMENTED** [review]
Please,  remember that don't need to include the debug logs with your commits

File: src/aura/projectDetailsItem/projectDetailsItem.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.nameClass.THIS {
```
Comment (Olaznog0) [inline src/aura/projectDetailsItem/projectDetailsItem.css]: Please remove unused class projectDetailsItem.css

File: src/aura/stepCard/stepCard.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.THIS {
```
Comment (Olaznog0) [inline src/aura/stepCard/stepCard.css]: Please remove unused class stepCard.css

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+config/
+*.sublime-project
```
Comment (Olaznog0) [inline .gitignore]: Please try not to have an extra space/newline at end-of-file code.

**Pull request closed without merging**